### PR TITLE
[NUI] Fix issue that objects are disposed outside callback.

### PR DESCRIPTION
**Pull request by huayongxu**
### Description of Change ###

1) Renaming some private variables.

### API Changes ###

 - ACR:

### CODE REVIEW

File: src/Tizen.NUI/src/internal/WebView/WebView.cs [hunk 1]
```diff
@@ -2008,9 +2018,12 @@ private void OnFrameRendered()
...
         {
-            ImageView image = new ImageView(data, true);
-            screenshotAcquiredCallback?.Invoke(image);
-            image.Dispose();
+            if (screenshotImage != null)
+            {
+                screenshotImage.Dispose();
+            }
+            screenshotImage = new ImageView(data, true);
+            screenshotAcquiredCallback?.Invoke(screenshotImage);
```
Comment (myroot) [inline src/Tizen.NUI/src/internal/WebView/WebView.cs:2026]: I thinks you should not dispose screenshotimage even though warning.
warning. A lifetime of captured image should be decided by API caller

```suggestion
            screenshotAcquiredCallback?.Invoke(new ImageView(data, true););
```

**Comment by myroot** [comment]
I recommend to suppress warning

File: src/Tizen.NUI/src/internal/WebView/WebView.cs (hunk 1 above)
Comment (huayongxu) [reply src/Tizen.NUI/src/internal/WebView/WebView.cs]: Done.

**Review by myroot — APPROVED** [review]

File: src/Tizen.NUI/src/internal/WebView/WebHitTestResult.cs [hunk 2]
```diff
@@ -226,6 +226,7 @@ public ImageView Image
                 ImageView image = new ImageView(url.ToString());
                 image.Size = new Size(pixelData.GetWidth(), pixelData.GetHeight());
```
Comment (rabbitfor) [inline src/Tizen.NUI/src/internal/WebView/WebHitTestResult.cs:227]: Do we have to set `Size` value manually?
@hinohie Do you know about it?

**Review by AchoWang — APPROVED** [review]
Looks good to me.

**Comment by xuelian-bai** [comment]
looks good to me

File: src/Tizen.NUI/src/internal/WebView/WebHitTestResult.cs (hunk 2 above)
Comment (rabbitfor) [reply src/Tizen.NUI/src/internal/WebView/WebHitTestResult.cs:227]: (This comment is not related with this PR, so I'm going to merge this anyway)

**Review by zhouleonlei — APPROVED** [review]
LGTM

**Review by AdunFang — APPROVED** [review]
It is OK.

**Review by rabbitfor — APPROVED** [review]

File: src/Tizen.NUI/src/internal/WebView/WebHitTestResult.cs (hunk 2 above)
Comment (rabbitfor) [reply src/Tizen.NUI/src/internal/WebView/WebHitTestResult.cs:227]: I checked to the DALi team. Getting `Size` property right after create ImageView can return invalid size so it's better to set it manually.

Comment (hinohie) [reply src/Tizen.NUI/src/internal/WebView/WebHitTestResult.cs:227]: More detail / If we don't set `Size` manually, `View.WidthSpecification` series didn't setup (default as `WrapContent`).  If we set `Size` manually, `View.WidthSpecificatoin` setup fixed so most calculation at `layout` will be more light.

**Pull request merged**